sherpa-onnx/csrc/offline-paraformer-model.cc

```diff
@@ -23,6 +23,10 @@ class OfflineParaformerModel::Impl {
     sess_opts_.SetIntraOpNumThreads(config_.num_threads);
     sess_opts_.SetInterOpNumThreads(config_.num_threads);
 
+#ifdef ENABLE_GPU
```

Comment:
Could you prepend it with `SHERPA_ONNX`?
That is, use `SHERPA_ONNX_ENABLE_GPU`.

All macros in sherpa-onnx are expected to start with `SHERPA_ONNX_`